playbooks/ORCHESTRATION.md

```diff
@@ -64,11 +64,7 @@ Like mentioned in [Key configuration](#key-configuration), each root key corresp
 
 ### Adding a Root Key
 
-Ask the new root keyholder to run:
-```bash
-export GITHUB_USER=${GITHUB_USER}
-./scripts/step-0.sh && ./scripts/step-1.sh
-```
+Instruct any new root keyholder to follow [Registering a new root key](#registering-a-new-root-key)
```

Comment:
```suggestion
Instruct any new root keyholder to follow [Registering a new root key](../KEYHOLDER.md#registering-a-new-root-key)
```
```suggestion
Instruct any new root keyholder to follow [Registering a new root key](#registering-a-new-root-key)
```

Reply:
I use these local refs a lot -- is it bad practice?